lib/util/http/azure.spec.ts

```diff
@@ -0,0 +1,31 @@
+import * as httpMock from '../../../test/http-mock';
+import { AzureHttp } from './azure';
+
+const azureApiHost = 'https://dev.azure.com';
+
+describe('util/http/azure', () => {
+  let azureApi: AzureHttp;
+
+  beforeEach(() => {
+    azureApi = new AzureHttp();
+  });
+
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  it('gets paginated json', async () => {
```

Comment:
```suggestion
  it('gets paginated JSON', async () => {
```

Abbreviations should be capitalized.